apps/emqx/src/emqx_broker_helper.erl

```diff
@@ -146,6 +150,8 @@ table_size(Tab) when is_atom(Tab) -> ets:info(Tab, size).
 %%--------------------------------------------------------------------
 
 init([]) ->
+    process_flag(priority, high),
```

Comment:
Nit. I'd propose to put that behind a configuration flag, feels a bit risky to me to distort the scheduling like that. Or maybe introduce a pool of helpers?

Reply:
We can maybe remove this flag for helper process, only set it for emqx_cm.
from the observations, it is mostly always on top when sort by reductions.

Reply:
Right. Also, I'm thinking all the other improvements (off-heap, no pmon, no selective receive) might be already enough to severely decrease the average reductions and memory management cost per disconnect?

IIUC `high` priority should not change a lot when CPU is not completely utilized, but when it is, cleaning up disconnects will take disproportionate priority over all other tasks, and the less there are CPU cores the more pronounced the effect will be. I guess we could consider this is better than accumulating huge message queue, but it doesn't actually guarantee that it stays bounded. For that I guess we'd need to introduce some form of backpressure during `register_sub`?